### ISSUE TITLE
rust: cargo fails to build on Debian 7.4

### ISSUE THREAD

**Issue by pramodhegde** [Linuxbrew/linuxbrew#412]
```
%brew install rust
==> Downloading https://static.rust-lang.org/dist/rustc-1.0.0-src.tar.gz
######################################################################## 100.0%
==> ./configure --prefix=/home/user/.linuxbrew/Cellar/rust/1.0.0 --release-channel=stable
==> make
==> make install
==> Cloning https://github.com/rust-lang/cargo.git
Updating /home/user/.cache/Homebrew/rust--cargo--git
Synchronizing submodule url for 'test/rust-installer-v1'
Synchronizing submodule url for 'test/rust-installer-v2'
Synchronizing submodule url for 'test/rust-installer-v1'
==> Checking out revision a48358155c90467ed9c897930dd0da4614605dac
==> Downloading https://static-rust-lang-org.s3.amazonaws.com/dist/2015-05-14/rustc-nightly-x86_64-apple-darwin.tar.gz
######################################################################## 100.0%
==> ./install.sh --prefix=/tmp/rust--cargo20150521-24860-1hswv02/rustc
==> Downloading https://static-rust-lang-org.s3.amazonaws.com/cargo-dist/2015-05-14/cargo-nightly-x86_64-apple-darwin.tar.gz
######################################################################## 100.0%
==> ./install.sh --prefix=/tmp/rust--cargo20150521-24860-1hswv02/target/snapshot/cargo
==> ./configure --prefix=/home/user/.linuxbrew/Cellar/rust/1.0.0 --local-rust-root=/tmp/rust--cargo20150521-24860-1hswv02/rustc
configure: 
configure: CFG_PREFIX           := /home/user/.linuxbrew/C ...
configure: CFG_LOCAL_RUST_ROOT  := /tmp/rust--cargo20150521-24860-1hsw ...
/tmp/rust--cargo20150521-24860-1hswv02/rustc/bin/rustc: 7: /tmp/rust--cargo20150521-24860-1hswv02/rustc/bin/rustc: Syntax error: Unterminated quoted string
configure: error: failed to run rustc at: /tmp/rust--cargo20150521-24860-1hswv02/rustc
```

Gist: https://gist.github.com/anonymous/f01adc7faf51bfac4b87

**Comment by sjackman** [comment]
Hi, Pramod. I'm afraid that I don't have the time to troubleshoot individual formula. If you're able to find a fix to this issue, please submit a PR.

**Comment by pramodhegde** [comment]
Will take a look at it early next week, if anyone hasn't got into it by then.

**Comment by rwhogg** [comment]
@pramodhegde  new versions of Rust/Cargo were committed [here](https://github.com/Linuxbrew/linuxbrew/commit/f939575005011d0681227aa530552d264edd933a#diff-5228b5cf14fdcdd1f7e2c1fd8091ad86).

Does the problem still occur for you?

**Comment by sjackman** [comment]
I've just merged Homebrew into Linuxbrew, which included a change to this formula. Did it resolve your issue?

**Comment by sjackman** [comment]
This issue may be resolved by #846.
